tests/unit/components/Messages/Messages.test.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { render, waitFor } from "@testing-library/react-native";
+import { NavigationContainer } from "@react-navigation/native";
+
+import factory from "../../../factory";
+import Messages from "../../../../src/components/Messages/Messages";
+
+const mockedNavigate = jest.fn( );
+const mockMessage = factory( "RemoteMessage" );
+
+jest.mock( "../../../../src/components/Messages/hooks/useMessages" , ( ) => ( {
+  __esModule: true,
+  default: ( ) => {
+    return {
+      messageList: [mockMessage],
```

Comment:
This should also be `messages`, not `messageList`, since that's what is being returned from the hook.